SteamGameCompare.py

```diff
@@ -239,19 +280,23 @@ def determineProperList(game):
 
 # This is for console output and will, likely, be removed eventually
```

Comment:
I like that there is a console only interface. I think that this is appealing to a niche group that like to do stuff with wget and curl. Could be interesting to serve up shell scripts over http that are connected to the api.

Reply:
Noted. Likely it won't be in this format though. Maybe just provide a console-output only version as a separate script